docs/guide/snaps.md

```diff
@@ -38,36 +38,36 @@ There are a number of simple snaps available for you to learn from and begin to
 
 ### Getting Flask running in your local environment
 
-[MetaMask Flask](https://metamask.io/flask) is a distribution channel for developmental, experimental, kooky stuff for MetaMask, and Snaps is the first feature to be released through it.
+[MetaMask Flask](https://metamask.io/flask) is a distribution channel for developmental, experimental, and kooky stuff for MetaMask. Snaps is just the first feature to be released through it.
 
-- Start by downloading [Flask](https://metamask.io/flask) and adding it to the available browser of your choice
+- Start by downloading [Flask](https://metamask.io/flask) and adding it to one of the available browsers of choice
 
-At this point, you should see Flask added to your browser as an extension. If you currently have another build of MetaMask installed in your browser, disable it (but make sure you have your Secret Recovery Phrase backed up!).
+After adding Flask as a browser extension, if you currently have another build of MetaMask installed in your browser, disable it (ensure you have your Secret Recovery Phrase backed up!).
 
-### Serving a snap to your local environment
+### Serving a Snap to Your Local Environment
 
-Now that you have Flask in your browser, it's time to find something to plug into it. If you don't want to download any code for the moment, check out the example snap [deployed here](https://metamask.github.io/snap-template/) and follow the instructions below under 'Interacting with the Snap'.
+With Flask in your browser, it's time to plug something into it. If you don't want to download any code for the moment, check out the example snap [deployed here](https://metamask.github.io/snap-template/) and follow the instructions below under 'Interacting with the Snap'.
 
 If you're ready to jump in and see how a Snap looks on the backend, follow the steps below!
 
 Open up a new terminal window, and do the following:
 
 - clone the snaps monorepo [here](https://github.com/MetaMask/snaps-skunkworks)
-- `cd` into the repo, and run `nvm use 14.15` to make sure you're using the right version of nodejs
+- `cd` into the repo, and run `nvm use 14.15` our reccomended version of nodejs
```

Comment:
Slight typo there, it should be "recommended". Can the line be worded better? Feels like the comma should be placed elsewhere.

Reply:
```suggestion
- `cd` into the repo, and run `nvm use 14.15` our recommended version of nodejs
```

Reply:
How does this sound?


```suggestion
- `cd` into the repo, and then run `nvm use 14.15` to switch to our recommended version of Node.js
```

Reply:
Definitely better